### ISSUE TITLE
Error on webpack build with UglifyJs after adding eslint-plugin-compat 

### ISSUE THREAD

**Issue by escapedcat** [amilajack/eslint-plugin-compat#144]
After adding the plugin I get this error message during a webpack build:
```
ERROR in foo.js from UglifyJs
Unexpected token punc «{», expected punc «(» [foo.vue:306,0][foo.js:1438,16]
```

In this case it's this piece of code:
```
try {
  ...
} catch {
  ...
}
```

If I change it to this it works:
```
try {
  ...
} catch(e) {
  ...
}
```

Without adding this plugin the build works.  
Not sure what this is about. If needed/wanted I can provide more info.

**Comment by vikr01** [comment]
When you say "adding this plugin" do you mean installing it, or adding it to your `eslint` config?

Are you using `babel` v6? I think this is related to `@babel/runtime` being a dependency (though it doesn't seem to be used).

cc @amilajack

**Comment by escapedcat** [comment]
I'm sorry! It's not this plugin!

The error starts as soon as I add a `.browserlistrc` file. I this case with this content:
```
last 2 Chrome versions
last 2 Firefox versions
```

Looks like I need to do some digging regarding UglifyJs and browserlist.  
Sorry for the false alarm!

**Comment by escapedcat** [comment]
Updating `uglifyjs-webpack-plugin` to latest version (i.e. `2.0.1`) helped.  
[This post on SO was useful](https://stackoverflow.com/a/46294198/1667461).  
No idea how adding a `.browserlistrc` file is related to `UglifyJs`. 
¯\_(ツ)_/¯
